package.json

```diff
@@ -83,7 +84,16 @@
     "test:examples": "npm run test:examples:regular && npm run test:examples:websites",
     "test:examples:regular": "cross-env CI=true jest ./examples --testPathIgnorePatterns ./examples/integrate-with-react",
     "test:examples:websites": "cd ./examples/integrate-with-react && npm i && npm run test",
-    "test:blackbox": "cross-env CI=true jest ./test/blackbox",
+    "test:blackbox": "concurrently --group -n csharp,go,java,javascript,python,rust,typescript \"npm run test:blackbox:csharp\" \"npm run test:blackbox:go\" \"npm run test:blackbox:java\" \"npm run test:blackbox:javascript\" \"npm run test:blackbox:python\" \"npm run test:blackbox:rust\" \"npm run test:blackbox:typescript\"",
+    "test:blackbox:csharp": "cross-env CI=true jest ./test/blackbox/blackbox-csharp",
+    "test:blackbox:go": "cross-env CI=true jest ./test/blackbox/blackbox-go",
+    "test:blackbox:java": "cross-env CI=true jest ./test/blackbox/blackbox-java",
+    "test:blackbox:javascript": "cross-env CI=true jest ./test/blackbox/blackbox-javascript",
+    "test:blackbox:python": "cross-env CI=true jest ./test/blackbox/blackbox-python",
+    "test:blackbox:rust": "cross-env CI=true jest ./test/blackbox/blackbox-rust",
```

Comment:
:bowing_woman: Splitting these tests is smart! Thanks for doing this. I'm going to start working on the Rust black-box tests in a branch of `jonaslagoni:feature/fix_all_blackbox_tests`

Reply:
Awesome 🔥 I am gonna mark rust as skipping them for now, and then you can enable them on your own 🙂

Reply:
I just pushed the rest of the current changes I have.

The last issues I found for rust, was something to do with enums and unions, where if an enum model was referenced in a union it expected the model to implement some eq something function I think 🤔